examples/fast_neural_style/README.md

```diff
@@ -0,0 +1,74 @@
+# fast-neural-style
+
+### Introduction
+This example is ported over from [pytorch-examples](https://github.com/pytorch/examples/tree/master/fast_neural_style).
+
+It uses pytorch-ignite to implement an algorithm for artistic style transfer as described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155).
+
+### Requirements
+
+The code can be executed using both `Python 2.7` and `Python 3.5`.
+
+* `torch`
+* `torchvision`
+* `pytorch-ignite`
```

Comment:
should be `ignite` as it's the package name`

Reply:
Incorporated